DQM/BeamMonitor/plugins/BeamMonitor.cc

```diff
@@ -661,6 +664,12 @@ void BeamMonitor::beginLuminosityBlock(const LuminosityBlock& lumiSeg, const Eve
 
 // ----------------------------------------------------------
 void BeamMonitor::analyze(const Event& iEvent, const EventSetup& iSetup) {
+  Handle<TCDSRecord> tcdsData;
+  iEvent.getByToken(tcdsToken_, tcdsData);
```

Comment:
```suggestion
  const TCDSRecord& tcdsData = iEvent.get(tcdsToken_);
```
as per https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideEDMGetDataFromEvent#get